### DISCUSSION TITLE
Add Farsi/persian language to lokalise

### DISCUSSION THREAD

**Discussion by danyhm** [home-assistant/frontend#9456] [Localization]
Hello,

I've recently started using HA and the Farsi language is missing for users speaking that language.

please add the Farsi language to both the frontend and the android/IOS app so I can start translating the strings.

Thanks

**Comment by dshokouhi** [comment]
I can help out with the android repo....make sure to file a separate request with iOS please. Someone from the frontend team will need to fill this request there.

**Reply by dshokouhi** [reply]
@danyhm what is your username on Lokalise?

**Reply by Gaygysyz05** [reply]
Hello. You can add Turkmen language too. I will be glad if you answer
